.k8s/deploy-k8s.groovy

```diff
@@ -0,0 +1,67 @@
+//Groovy scripts throw lots of WARNING messages, but it is fine
+//https://issues.apache.org/jira/browse/GROOVY-8339
+
+
+def DOCKER_REG = System.getenv("DOCKER_REGISTRY")
```

Comment:
Might want to tell people to set `DOCKER_REGISTRY` and `DOCKER_WRAPPER` in the README. If it isn't already in some other README. If you do, let people know to use the full path to the docker wrapper file.

Reply:
We actually won't need the wrapper, at least for now. I'll update the readme to mention the env variable.